src/Feersum/Compile.fs

```diff
@@ -880,27 +963,32 @@ let compileFile options (output: string) (source: string) =
     if Diagnostics.hasErrors diagnostics then
         diagnostics
     else
-        let symbols =
+
+        // Open the output streams. We don't use an `Option` directly here for
+        // the symbols stream so we can drop it with `use`.
+        use outputStream = File.OpenWrite output
+        use symbols =
```

Comment:
Having to dispose this stream here before we can compile a dependant lib points towards us reading the symbols when we do the mono reflection to compute the library signatures. We definitely don’t need to be doing that.